src/main/java/randoop/contract/EqualsTransitive.java

```diff
@@ -0,0 +1,62 @@
+package randoop.contract;
+
+import randoop.Globals;
+
+/**
+ * The contract:
+ * <code>!(x0.equals(x1) &amp;&amp; x1.equals(x2))||x0.equals(x2)</code>.
```

Comment:
You can use the HTML code for a right arrow: &#8594;
The way to literally get "->" is to use "-&gt;"